layouts/partials/related.html

```diff
@@ -0,0 +1,31 @@
+{{ $pageType := .pageType | default .index }}
+{{ $related := where (.page.Site.RegularPages.RelatedIndices .page .index) ".Page.Type" $pageType }}
+{{ if gt (len $related) 0 }}
+<h3 id="o-related__title">{{ .title }}</h3>
```

Comment:
Pass further identifier to the related partial. Otherwise the id for the ul is not unique.